Task11/Catalog.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Task11
+{

```

Comment:
Create your class Catalog - is a collection, like a dictionary, that stores books-> at least IEnumarable has to be implemented, plus indexer